request/builder.js

```diff
@@ -580,6 +583,24 @@ class ContractExecutionRequestBuilder {
     return this;
   }
 
+  /**
+   * @param {boolean} useFunctionIds
+   * @return {ContractExecutionRequestBuilder}
+   */
+  withUseFunctionIds(useFunctionIds) {
+    this.useFunctionIds = useFunctionIds;
+    return this;
+  }
+
+  /**
+   * @param {string[]} functionIds
+   * @return {ContractExecutionRequestBuilder}
+   */
+  withFunctionIds(functionIds) {
+    this.functionIds = functionIds;
+    return this;
+  }
+
```

Comment:
These new functions seem not to be used anywhere currently. Is this PR a part of adding new contract interfaces for DL v3.5.x?

Reply:
@jnmt 
Thank you for your question.
Yes, this PR is part of the upgrade for supporting DL v3.5.x.
I tried to split changes into small PR.
I hope they can be easier to review.

Reply:
@supl Thanks. I see.